### ISSUE TITLE
[Internal] Kill our custom baseline textview

### ISSUE THREAD

**Issue by SUPERCILEX** [firebase/FirebaseUI-Android#1464]
This went through: https://github.com/material-components/material-components-android/pull/56

Should be done when we upgrade to support lib v28 after `v4.2.1`.

**Comment by SUPERCILEX** [comment]
Here's the class: https://github.com/firebase/FirebaseUI-Android/blob/8bda799672d44fd2994d2498441859fae3ccb530/auth/src/main/java/com/firebase/ui/auth/util/ui/BaselineTextInputLayout.java

**Comment by samtstern** [comment]
@SUPERCILEX you're everywhere!  SGTM

**Comment by samtstern** [comment]
Upgrading to compile SDK 28 gave me some weird build issues, so will deal with this in 4.3.0 (which will come soon after 4.2.1).

**Comment by samtstern** [comment]
This has been released in version `4.3.0`.

**Comment by samtstern** [comment]
Looking at `TextInputLayout.class` in the `28.0.0` release of the design support library, I don't think your change made it in :-/

**Comment by SUPERCILEX** [comment]
Huh, I guess it's only available in AndroidX. Maybe it's time to do the migration?

**Comment by samtstern** [comment]
Yeah one of many reasons!  I guess we have to just swallow that pill soon.
